Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. Describe some of the differences between .forEach & .map. 
+
+The foreach method is a built in callback function that iterates through an array or object to simply return all of the values. It's used to see what all is in the array or object, it doesn't necessarily return the items in a desirable format. The map function is a built in function that iterates through an array to replace existing values with new ones. For example, each place holder within the old array will have a place holder for an item in the new array. You can iterate through values with a math funciton or change the actual values any number or ways. If you only change some of the values, some of the old values can remain in the new array.  
```

Comment:
Yeah, exactly! Another thing is that forEach doesn't return anything whereas map does. It's already implied in your explanation. 😄